ups/meas_base.cfg

```diff
@@ -21,5 +21,5 @@ config = lsst.sconsUtils.Configuration(
     __file__,
     headers=["lsst/meas/base.h"],
     hasDoxygenInclude=False,
-    hasSwigFiles=True,
+    hasSwigFiles=False,
```

Comment:
Ouch. Best to leave it, I guess.

We can take a cleanup pass after we switch to pybind11 (and presumably can change this default to false).